sdk/typescript/src/types/common.ts

```diff
@@ -9,3 +9,35 @@ export type ObjectOwner =
   | { ObjectOwner: SuiAddress }
   | 'Shared'
   | 'Immutable';
+
+
+const TX_DIGEST_BASE64_LENGTH = 44;
```

Comment:
```suggestion
// source of truth is
// https://github.com/MystenLabs/sui/blob/acb2b97ae21f47600e05b0d28127d88d0725561d/crates/sui-types/src/base_types.rs#L171
const TX_DIGEST_LENGTH = 32;
```

I think 44 can work but I would love to directly use the source of truth definition here